tractor/_runtime.py

```diff
@@ -826,7 +829,12 @@ async def _push_result(
 
             if ctx._backpressure:
                 log.warning(text)
-                await send_chan.send(msg)
+                try:
+                    await send_chan.send(msg)
```

Comment:
If using backpressure on  a stream we need to be sure we don't crash when the local feed memchan has been broken.

Need a test for this, though I can't remember if this was the original issue that caused the `ipc_failure_during_stream.py` example to inf hang before?